### DISCUSSION TITLE
Community Chat

### DISCUSSION THREAD

**Discussion by auronandace** [zellij-org/zellij#347] [General]
I saw this project from This Week in Rust and I must say I am very impressed. It is really good to have an alternative to tmux that is written in Rust. Well done to all the hard work everyone has put in and no doubt it is only going to get better.

I was just wondering what is the reasoning behind using discord as the place to chat? Is it just the potential user-base? Have you considered using a more open platform like Matrix, Mattermost or Zulip? My personal preference would be an IRC channel on Freenode. Matrix can bridge to IRC.

Is the chat for anyone to join or is it more developer centric?

I hope my questions don't come across as a criticism and I fully understand that my preferences would mean very little as I'm not a developer and no doubt you all want to do what is in the best interest and convenience for everyone involved. I just think if more people used open source alternatives for chat platforms then it is better for everyone involved (and helps avoid supporting closed source software).

**Comment by auronandace** [comment]
I wrote the above before Freenode changed. Libera Chat is now what Freenode once was but better.

Also wanted to say that I'm happy there is now a Matrix room available. Thank you for such a great project!

**Reply by tsdh** [reply]
I also would like to express my concerns that a FOSS software's community is locked in a proprietary walled garden like discord.

What's the matrix room you are talking about?  I haven't been able to find another community except the discord...

**Reply by a-kenji** [reply]
It is one of the first links of the README.md , right after the discord link.

**Reply by tsdh** [reply]
It took me another 5 minutes staring at https://github.com/zellij-org/zellij/blob/main/README.md#how-do-i-get-involved and looking for matrix right after the "chat" discord link.  But now I found the badge at the very top of the page. :-)

**Comment by tsdh** [comment]
Yeah, at least an inofficial `##zellij` on libera.chat would be great (similar to `##rust`).
